x/multisig/abci.go

```diff
@@ -26,14 +26,14 @@ func handleKeygens(ctx sdk.Context, k types.Keeper, rewarder types.Rewarder) {
 	for _, keygen := range k.GetKeygenSessionsByExpiry(ctx, ctx.BlockHeight()) {
 		k.DeleteKeygenSession(ctx, keygen.GetKeyID())
 
+		slices.ForEach(keygen.GetMissingParticipants(), rewarder.GetPool(ctx, types.ModuleName).ClearRewards)
```

Comment:
```suggestion
                pool := rewarder.GetPool(ctx, types.ModuleName)
		slices.ForEach(keygen.GetMissingParticipants(), pool.ClearRewards)
```
so that you don't have to get it again below. Same for sig handling.